ruby/lib/core/sender.rb

```diff
@@ -29,10 +29,17 @@ class Sender < Link
 
     # Open the {Sender} link
     #
+    # @note Sender **auto-settlement** only occurs for a delivery after the
+    #     sender receives a settled disposition for that delivery. Otherwise,
+    #     there would be no way to receive any further events for that delivery
+    #     (such as the subsequent on_settle message that might be expected when
+    #     the receiver finally settles the message).
     # @overload open_sender(address)
     #   @param address [String] address of the target to send to
     # @overload open_sender(opts)
-    #   @option opts [Boolean] :auto_settle (true) if true, automatically settle transfers
+    #   @option opts [Boolean] :auto_settle (true) if true, automatically settle transfers.
+    #       **Note:** Sender auto-settlement only occurs for a delivery after the sender receives
+    #       a settled disposition for that delivery.
```

Comment:
As above we're just describing the basic behaviour, do inline as I suggested for python.

Reply:
Done.